src/include/block_chunkcache.h

```diff
@@ -68,6 +69,7 @@ struct __wt_chunkcache {
     char *dev_path;             /* the storage path if we are on a file system or a block device */
     unsigned int evict_trigger; /* When this percent of cache is full, we trigger eviction. */
     unsigned int hashtable_size;
-    int type;              /* location of the chunk cache (volatile memory or file) */
+    char *memory;          /* memory location for the assigned chunk space */
     char **pinned_objects; /* list of objects we wish to pin in chunk cache */
```

Comment:
@korteland Should the content in this struct be in a specific order?

Reply:
I've put it in alphabetical order as per the struct prior.

Reply:
I would be more inclined to do a logical ordering or ordering to minimise padding than alphabetical.

Reply:
I think its okay the way it is - that is more of a stylistic choice.